sensor_msgs/Image image
sensor_msgs/CameraInfo camInfo
---
std_msgs/Bool found_top